proto/rpc.proto

```diff
@@ -8,7 +8,7 @@ service KV {
   // Range gets the keys in the range from the key-value store.
```

Comment:
I will take these updates in a day or two; I want to update the proto files and add the appropriate new methods on our builders concurrently.